src/View/ViewVarsTrait.php

```diff
@@ -65,6 +66,7 @@ public function getView($viewClass = null)
         }
 
         if ($this->View && $this->View instanceof $className) {
+            $this->View->viewVars = $this->viewVars;
```

Comment:
@ADmad Ok.

@leighmackay I was thinking about layout, view, and helpers. But ADmad clarified that they are not easily synced.